pocs/camera/fli.py

```diff
@@ -32,6 +32,7 @@ def __init__(self,
                  name='FLI Camera',
                  set_point=25 * u.Celsius,
                  filter_type='M',
+                 library_path=False,
```

Comment:
This FLI camera doesn't have a docstring so things like the `library_path` aren't documented anywhere.

Reply:
Huh. None of the camera classes have docstrings. How did I get away with that up to now?